--- a/tools/server/server-context.cpp
+++ b/tools/server/server-context.cpp
@@ -4460,7 +4460,9 @@ static json get_res_model_info(const server_context_meta & meta) {
             {"vocab_type",  meta.model_vocab_type},
             {"n_vocab",     meta.model_vocab_n_tokens},
             {"n_ctx",       meta.slot_n_ctx},
+            {"loaded_context_length",  meta.slot_n_ctx},
+            {"max_context_length",     meta.model_n_ctx_train},
             {"n_ctx_train", meta.model_n_ctx_train},
             {"n_embd",      meta.model_n_embd_inp},
             {"n_params",    meta.model_n_params},
